scripts/install_uWebSockets.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
```

Comment:
What is the purpose of this script? Why not insert this code directly into the Dockerfiles?